libenv/unix_iface.c

```diff
@@ -1217,6 +1258,28 @@ void GetNetworkingInfo(EvalContext *ctx)
                     default_route = route;
                 }
             }
+
+            /* fixup MTU -- use the interface's MTU if the route doesn't specify
+             * its own [CFE-3098] */
+            JsonElement *mtu = JsonObjectGet(route, "mtu");
+            if (mtu != NULL &&
+                JsonGetElementType(mtu) == JSON_ELEMENT_TYPE_PRIMITIVE &&
```

Comment:
I'd prefer the `&&` operator on the start of lines here.

Reply:
There's a very similar block of code right above this new one I added which uses this style. Maybe the whole file could be reformatted with clang-format in a separate commit, but I'm not using it.